pkg/api/runner.go

```diff
@@ -94,3 +94,9 @@ type CollectionInput struct {
 	// plan manifest, coalesced with any user-provided overrides.
 	RunnerConfig interface{}
 }
+
+// Healthcheckable is the interface to be implemented by a runner that can be
+// healthcheckable.
+type Healthcheckable interface {
```

Comment:
nitpick, but in Go, interfaces that have a method `Do`, are usually called `Doer`, so this should probably be named `Healthchecker`, so that when you see the interface name, you know there is a `Healthcheck` method :)

Reply:
I think that's a great nitpick though haha. Gonna change.